.github/workflows/build.yml

```diff
@@ -0,0 +1,89 @@
+name: Build wheels and deploy
+
+on:
+  workflow_dispatch:
+    inputs:
+      confirm_ref:
+        description: "Enter branch/tag name to deploy to PyPI, or leave blank to only build wheels."
+        default: ""
+      override_version:
+        description: "Override version number:"
+        default: ""
+
+
+jobs:
+  deploy_canary:
+    name: "Will deploy to PyPI if ticked"
+    runs-on: ubuntu-18.04
+    if: endsWith(github.ref, format('/{0}', github.event.inputs.confirm_ref))
+    steps:
+      - name: "Built wheels will be deployed at the end."
+        run: |
+          echo "Built wheels will be deployed at the end."
+
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, windows-latest, macos-latest]
```

Comment:
is there a reason why this is not ubuntu-latest?

Reply:
I just got it from the `cibuildwheel` default action - it makes no difference at all, because all the actual build is done within a Docker image.  I suspect 18.04 was the last LTS release of Ubuntu.

Reply:
I turned this into `ubuntu-latest` because I suppose `ubuntu-latest` will always work, whereas `ubuntu-18.04` will (eventually) go out-of-date.